modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/BaseNotificationTypeTest.java

```diff
@@ -149,6 +149,42 @@ public static void setUpClass() throws Exception {
 			"textObjectField", RandomTestUtil.randomString()
 		).build();
 
+		siteScopeObjectEntryValues = LinkedHashMapBuilder.<String, Object>put(
```

Comment:
Hey @rijema, rename to `scopeSiteObjectEntryValues` to follow the pattern ([see](https://github.com/liferay/liferay-portal/blob/f6492225a350aa32a5de5715acb8c780f2069c12/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java#L315))

please rename all other places in these changes that is using siteScope as prefix as well.

Thanks!